beta-src/src/state/game/game-api-slice.ts

```diff
@@ -323,6 +334,26 @@ const gameApiSlice = createSlice({
     updateTerritoriesMeta(state, action) {
       state.territoriesMeta = action.payload;
     },
+    processUnitDoubleClick(state, clickData) {
+      const {
+        order,
+        data: {
+          data: { contextVars },
+        },
+      } = current(state);
+      if (contextVars?.context?.orderStatus) {
+        const orderStates = getOrderStates(contextVars?.context?.orderStatus);
+        if (orderStates.Ready) {
+          return;
+        }
+      }
+      const { inProgress } = order;
+      if (inProgress) {
+        // double click move in progress
+      } else {
+        startNewOrder(state, clickData);
+      }
```

Comment:
ternary
`inProgress ? : startNewOrder(state, clickData)'`

Reply:
This is still in progress, those are just placeholders.